web.browser/src/containers/PostList/PostListContainer.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { updateVote, sortPopular, sortNewest } from '../../redux/actions';
+import PostList from './PostList';
+
+import './styles.css';
+
+class PostListContainer extends Component {
+  // constructor() {
+  //   super();
+
+  //   this.updateVote = this.updateVote.bind(this);
+  //   this.sortPopular = this.sortPopular.bind(this);
+  //   this.sortNewest = this.sortNewest.bind(this);
+
+  //   this.state={
+  //     posts: data.posts,
+  //     orderby: 'newest'
+  //   }
+  // }
+
+  updateVote(post) {
+    const upVote = post.votes + 1;
```

Comment:
This logic should happen in your reducer. You send the `UP_VOTE`message, and in the reducer you handle the incrementation.